src/index.ts

```diff
@@ -17,15 +22,31 @@ const env = {
   location: process.env.ORGANIZATION ?? 'sarcophagus.aragonid.eth',
 }
 
+function generateVoteId(id: string): any {
+  let aragonVoteIdPrefix = "appAddress:0xf483c1f7858dd19915d0689d26cb3487fc90b640-vote:"
+  let aragonVoteId = aragonVoteIdPrefix + "0x" + Number(Number(id).toString(16)).toString(16)
+  return aragonVoteId
+}
+
 async function main() {
   const org = await connect(env.location, 'thegraph', { network: env.network })
   const voting = await connectVoting(org.app('voting'))
   const votes = await voting.votes({ first: 100 })
+
+  let voteId = generateVoteId(process.env.VOTE_ID)
+
+  const vote = votes.filter((v: any) => {
+    return v.id === voteId;
+  })
```

Comment:
This would be better as a `find` rather than a `filter`, since we only want it to return one record.

Then we would want to make sure `vote` isn't `undefined` --

```
if (!vote) {
   throw Error("Unable to retrieve vote: ${voteId}")
}
```

Then the `voteWithCasts` could be refactored to use the single vote. Not sure the best way to do this, but one way:

```
const casts = await vote.casts()

...

printVotes([{...vote, casts}])
```

Reply:
@sethhrbek - All changes have been made and pushed, please re-review when you get the chance